lib/src/configuration.cc

```diff
@@ -292,6 +307,8 @@ const Configuration::Agent& Configuration::getAgentConfiguration() const
 
 void Configuration::reopenLogfile() const
 {
+    // TODO(ale): consider reopening PCP access file
```

Comment:
I think I want to look into this TODO. Also busy at PuppetConf. Otherwise no issues.